src/postgres/commands/createPostgresQuery/validateIdentifier.ts

```diff
@@ -21,5 +21,90 @@ export function validateIdentifier(identifier: string): string | undefined {
         return localize('canOnlyContainCertainCharacters', 'Name can only contain letters, underscores, digits (0-9), and dollar signs ($).');
     }
 
+    if (reservedWords.has(identifier.toLowerCase())) {
+        return localize('cantIncludeReservedWord', 'Name cannot include reserved word "{0}".', identifier);
```

Comment:
When you say "include", I would assume that means "all" _and_ "all123" are disallowed. I don't think that's what you mean, though...

```suggestion
        return localize('cantIncludeReservedWord', 'Name cannot be reserved word "{0}".', identifier);
```